### ISSUE TITLE
Do we need new enum cases in `JSValue` for closures?

### ISSUE THREAD

**Issue by MaxDesiatov** [swiftwasm/JavaScriptKit#118]
Now that `JSClosure` and `JSOneshotClosure` are no longer subclasses of `JSFunction`, how are they supposed to be converted to `JSValue`? Should the `object` case be used?

**Comment by kateinoigakukun** [comment]
I think the `object` case should be used for this purpose because we don't need to distinguish object case and closure case internally. If you want API consistency, we can add a new static func instead of a new case.
```swift
static func closure(_ closure: JSClosure) -> JSValue { .object(closure) }
```

**Comment by MaxDesiatov** [comment]
Yeah, that would make sense, I can see people asking why there is `JSValue.function`, but no `JSValue.closure`
